src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HearingTypeHandler.java

```diff
@@ -76,7 +72,7 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
         boolean isAgeAssessment = appealTypeForDisplay == null && (asylumCase.read(AGE_ASSESSMENT, YesOrNo.class).equals(Optional.of(YES)));
```

Comment:
appealType != null && appeal NOT in detention

Reply:
`    if (callback.getEvent() == Event.START_APPEAL) {
            YesOrNo hearingTypeResult = isRpDcAda ? YES : NO;
            asylumCase.write(HEARING_TYPE_RESULT, hearingTypeResult);
        }`

I think this is the cleanest way of doing it. Works as expected.